ui/app/components/tx-list.js

```diff
@@ -97,16 +97,19 @@ TxList.prototype.renderTransactionListItem = function (transaction, conversionRa
     address,
     transactionAmount,
     transactionHash,
-    className: '.tx-list-item.tx-list-clickable',
+    className: '.tx-list-item',
     conversionRate,
   }
 
+  const clickableClassname = '.tx-list-clickable'
+
   if (transactionStatus === 'unapproved') {
     opts.onClick = () => showConfTxPage({id: transActionId})
-    opts.className += '.tx-list-pending-item-container'
+    opts.className += `${clickableClassname}.tx-list-pending-item-container`
```

Comment:
Chatted live. Let's look at cleaning this up by moving some of the conditional logic out into functions, get the render func under 50 lines if possible (it's now 60+).  Could be moved into a TODO/comment in a future issue, not hipri.

Reply:
...could use classnames for some of this.